addons/MultiAudio/src/style.css

```diff
@@ -19,6 +19,7 @@
 
 .multiaudio-selected {
     border: 1px solid orange;
+    z-index: 100;
```

Comment:
Are you sure about z-index? It will break a lesson UI if multile modules are stack.

Reply:
multiaudio-selected is only given to the element that is being dragged or has been selected by clicking on it. Without setting z-index, there are issues with dragged element being displayed behind other modules.